src/Mobs/PassiveMonster.h

```diff
@@ -60,6 +60,9 @@ class cPassiveMonster:
 	/** The monster's breeding partner. */
 	cPassiveMonster * m_LovePartner;
 
+	/** Remembers the player is was last fed by for statistics tracking */
+	cPlayer * m_Feeder;
```

Comment:
We shouldn't store a raw pointer to a cPlayer since the player may disconnect, could you store a UUID instead and be like cWorld::DoWithPlayerByUUID?

Reply:
Yes. I just need the stat manager of the player. The way it's obtained doesn't matter.

Reply:
fixed - added also the cow breeding achievement wile at it